core/include/debug.h

```diff
@@ -31,6 +31,7 @@
 
 #include <stdio.h>
 #include "sched.h"
+#include "thread.h"
```

Comment:
Maybe guard the includes in `#if DEVELHELP`? Otherwise removing `#include "debug.h"` will have side effects which might baffle a user.